{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -181,22 +227,39 @@ def get_var(name, default):
 EMAIL_SUPPORT = get_var('{{ cookiecutter.project_name|upper }}_SUPPORT_EMAIL', 'support@example.com')
 DEFAULT_FROM_EMAIL = get_var('{{ cookiecutter.project_name|upper }}_FROM_EMAIL', 'webmaster@localhost')
 
+MAILGUN_URL = get_var('MAILGUN_URL', 'https://api.mailgun.net/v3/micromasters.mit.edu')
```

Comment:
I left that in on purpose, despite its name we use it in bootcamp as well. @pdpinch just to confirm, it's fine to use it here as well?